sa/model.go

```diff
@@ -284,6 +300,17 @@ type regModel struct {
 	Status    string `db:"status"`
 }
 
+type certificateModel struct {
```

Comment:
Feels like this type should either be adjacent to `lintingCertModel`, or adjacent to where it is used in `SelectCertificate`.